### DISCUSSION TITLE
How would I go about using the affine transform from dipy to interpolate with SimpleItk?

### DISCUSSION THREAD

**Discussion by demcbs** [dipy/dipy#2714] [Registration]
Hiya,
I have run 3D image (MRI images) registration with dipy and the results look good. However, for me, it would be preferable for to use 'bspline' interpolation as supported by SimpleITK, so I would like to use the affine of this image transformation to interpolate the images with SimpleITK. 

I thought I'd be able to use the affine for this in a straightforward manner, by creating an affine transform in SimpleITK and setting this transform + the reference image in the resampler, but so far it doesn't work. I thought it might have something to do with the orientations, but I've played with it and did not get good results so far. Does anyone know how to do this?

**Comment by skoudoro** [comment]
Hi @demcbs,

You might have an answer via SimpleITK forum/discussion. Personnally, I do not know SimpleITK standard. If you could share where we can find this information in their doc. we might have a look. 

> I thought I'd be able to use the affine for this in a straightforward manner, by creating an affine transform in SimpleITK and setting this transform + the reference image in the resampler, but so far it doesn't work.

What does it mean "it doesn't work"? wrong results? application crash ? etc???
